### DISCUSSION TITLE
Nuxt 3 Plugins Auto Import Disabling

### DISCUSSION THREAD

**Discussion by razmatinyan** [nuxt/nuxt#23523] [Questions]
Hello. Is there any way to disable auto-import only for plugins from `/plugins` directory?

**Comment by memic84** [comment]
What are you trying to accomplish with disabling auto-import from the plugins directory? You want to load them conditionally?

**Reply by razmatinyan** [reply]
Yes, I wanted to load them conditionally and I want to know is it possible? I created `custom vue directive` and it is loading in whole website, but I want to load it when I want.

**Reply by memic84** [reply]
I don't think plugins are meant to be loaded in that way. You can choose to load then only on the client or server. Otherwise you should try to load the code not through plugins, but in your pages/component. It's hard to tell, without a code example.

**Reply by razmatinyan** [reply]
Okay, I just implemented that plugin functionality in `composables`, and it is better solution for my situation, because composables are not loading in whole app. I think I was just using plugins not correctly 😅.

**Thank you for your answer!**
